*: Create SECURITY.md (#25833)
pingcap/tidb: SECURITY.md

# Security Vulnerability Disclosure and Response Process

> As we all know, TiDB is a fast-growing open source database. In order to ensure its security, we have adopted the following security vulnerability disclosure and response process.

Reviewer: ```suggestion
TiDB is a fast-growing open source database. Therefore, a security vulnerability disclosure and response process is adopted to ensure the security of TiDB.
```

> The primary goal of this process is to reduce users’ total exposure time to publicly known vulnerabilities. To ensure TiDB products’ vulnerabilities can be fixed quickly, our security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.

Reviewer: ```suggestion
The primary goal of this process is to reduce the total exposure time of your data to publicly known vulnerabilities. To quickly fix TiDB products’ vulnerabilities, our security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
```

Reviewer 2: ```suggestion
The primary goal of this process is to reduce the total exposure time of users to publicly known vulnerabilities. To quickly fix vulnerabilities of TiDB products, the security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
```

> If you think you have found a vulnerability or have experienced security incidents involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).

Reviewer: ```suggestion
If you find a vulnerability or face security incidents involving vulnerabilities of TiDB products, you can report it as soon as possible to the TiDB security team (security@tidb.io).
```

Reviewer 2: ```suggestion
If you find a vulnerability or encounter a security incident involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).
```

Please kindly help provide as much vulnerability information as possible in the following format:

- Issue title*:

- Overview*:

- Affected components and version number*:

- CVE number (if any):

- Vulnerability verification process*:

- Contact information*:

*mean required

> # Response time

Reviewer: ```suggestion
# Response Time
```

> The TiDB security team will confirm the vulnerabilities within 2 working days after your submission and keep in touch with you.

Reviewer: ```suggestion
The TiDB security team will confirm the vulnerabilities and contact you within 2 working days after your submission.
```

> After a security vulnerability is fixed, we will thank the reporter publicly. However, in order to avoid negative impact, please kindly keep the vulnerability confidential before that. We would be really appreciated if you could obey the following code of conduct:

Reviewer: ```suggestion
After we fix the informed security vulnerability, we will thank you publicly. However, to avoid unnecessary troubles, keep the vulnerability confidential until we fix it. We would appreciate it if you could obey the following code of conduct:
```

Reviewer 2: ```suggestion
We will publicly thank you after fixing the informed security vulnerabilities. To avoid unnecessary troubles, please keep the vulnerability confidential until we fix it. We would appreciate it if you could obey the following code of conduct:
```

The vulnerability will not be disclosed until TiDB releases a patch for it.

> Do not disclose the details of the vulnerability, such as exploits code.

Reviewer: ```suggestion
The details of the vulnerability, for example, exploits code, will not be disclosed.
```
